app/assets/stylesheets/katello/katello.scss

```diff
@@ -8,12 +8,12 @@
 @import "katello/widgets/tabs";
 @import "katello/katello_sprites";
 
-@import "katello/jquery-ui-1.8.11.custom.css";
-@import "katello/jquery.loadmask.css";
-@import "katello/jquery.jnotify.css";
-@import "katello/jquery.treeTable.css";
-@import "katello/jquery.jscrollpane.css";
-@import "katello/ui.spinner.css";
+@import "jquery-ui-1.8.11.custom.css";
+@import "jquery.loadmask.css";
+@import "jquery.jnotify.css";
```

Comment:
Looks roughly like the same (https://github.com/theforeman/foreman/blob/develop/vendor/assets/stylesheets/jquery.jnotify.scss) which we can remove it here and from our code base if so. I am not even sure we actually use jnotify anywhere anymore.